server/movici_viewer/model/model.py

```diff
@@ -240,6 +243,25 @@ def get_scenarios(self):
         for name in self.iter_scenario_names():
             yield self.get_scenario(name)
 
+    def update_scenario(self, scenario: str, config_data: dict):
+        """Update scenario configuration file with new config data"""
+        import json
+        from ..exceptions import InvalidObject
+        
+        path = self.get_scenario_path(scenario)
+        
+        # Clear memoized cache for this scenario
+        if hasattr(self, '_memoize_cache'):
+            cache_key = f"get_scenario:{scenario}"
+            self._memoize_cache.pop(cache_key, None)
```

Comment:
I don't think we're caching this function, and if we do, I think we use a different mechanism. Where did you find this?